CMakeLists.txt

```diff
@@ -29,6 +29,13 @@ cmake_minimum_required(VERSION 2.6)
 project(gnuradio CXX C)
 enable_testing()
 
+# Apple only - look for libraries installed by macports
+if(APPLE)
+	SET(GCC_COVERAGE_COMPILE_FLAGS "-L/opt/local/lib -Qunused-arguments")
+	SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${GCC_COVERAGE_COMPILE_FLAGS}")
```

Comment:
(1) Adding a system-level library to the C* flags should be the last thing that happens, so that any internal-to-this-build library of the same name is found first. CMake tries as hard as it can to order search directories, but it's always better if we help CMake out where possible. Since we would be voluntarily adding in a search directory flag, it should be the last thing that's added to C*FLAGS.
(2) We should not hard-code search directories any more than required. For example, this addition will not benefit OS X Homebrew users by default, because brew installs exclusively into /usr/local ; ditto for Fink users, where software is installed into /sw/. Further, MacPorts can be instructed to install into other than /opt/local (I use /Developer/opt/local, to check for this issue when working on ports & 'port' internals). Thus your fix here is for your specific setup, and not a good general fix to whatever the issue is.